static/css/community.css

```
@@ -153,56 +182,56 @@ body.cid-community .community-section#events {
   display: none;
 }
 
+/* -------------------------------------
+ * Containers and blocks of sections - Community Values
+ ------------------------------------- */
 body.cid-community .community-section#values {
   width: 100vw;
   max-width: initial;
   background-image: url('/images/community/event-bg.jpg');
   color: #fff;
-  padding: 2em;
-  margin-top: 3em;
-}
-body.cid-community .community-section#values {
-  padding-left: calc((100vw - min(75vw,120em))/2);
-  padding-right: calc((100vw - min(75vw,120em))/2);
 }
 
+/* -------------------------------------
+ * Containers and blocks of sections - Meetups
+ ------------------------------------- */
 body.cid-community .community-section#meetups {
   width: 100vw;
```

Either this OR auto, As per your liking
```suggestion
  width: 100%;
```